ios/ThreadSafeQueue.swift

```diff
@@ -0,0 +1,28 @@
+class ThreadSafeQueue<T> {
+    private var array: [T] = []
+    private let queue = DispatchQueue(label: "com.stripe.threadSafeQueue")
```

Comment:
minor nit: queue naming
```suggestion
    private let queue = DispatchQueue(label: "com.stripe.terminal.reactnative.threadSafeQueue")
```